src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -123,31 +133,50 @@ public static void checkProperty(final HaxeFieldModel field, final AnnotationHol
       holder.createErrorAnnotation(field.getSetterPsi(), "Invalid setter accessor");
     }
 
-    if (field.getGetterType() == HaxeAccessorType.GET) {
-      final String methodName = "get_" + field.getName();
-      HaxeMethodModel method = field.getDeclaringClass().getMethod(methodName);
-      if (method == null) {
-        Annotation annotation = holder.createErrorAnnotation(field.getGetterPsi(), "Can't find method " + methodName);
-        annotation.registerFix(new HaxeFixer("Add method") {
-          @Override
-          public void run() {
-            field.getDeclaringClass().addMethod(methodName);
+    ResultHolder fieldType = field.getResultType();
+
+    final HaxeClassModel clazz = field.getDeclaringClass();
+
+    if (!clazz.isExternOrInterface()) {
+      if (field.getGetterType() == HaxeAccessorType.GET) {
+        final String methodName = "get_" + field.getName();
```

Comment:
I much prefer to have things like "get_" in a constant.  And these three lines generating a getter method name and detecting whether it exists should be a method on the field model.  Something like field.findGetterMethod, which would return null if it doesn't have one.